server/pom.xml

```diff
@@ -113,52 +113,307 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-shade-plugin</artifactId>
-                        <version>2.4.1</version>
+                        <groupId>org.codehaus.mojo</groupId>
```

Comment:
We should test whether it works in a distributed environment or not after removing the dependencies.